Santouka/css/main.css

```diff
@@ -0,0 +1,197 @@
+/* 共通 */
+body {
+  margin: 0;
+  background: url(../img/background2.jpeg) center center no-repeat;
+  width: 100%;
+  height: 52rem;
+  background-size: cover;
+  box-sizing: border-box;
+  background-color:rgba(255,255,255,0.5);
+  background-blend-mode: lighten;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+/*--------------------
+header
+--------------------*/
+header {
+  /* border: 2px solid red;  */
+  background-color:rgb(46, 45, 45);
+  display: flex;
+}
+
+header img {
+  /* border: 2px solid blue;  */
+  height: 4em;
+}
+
+header ul {
+  /* border: 2px solid blue;  */
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  /* justify-content:space-between; */
+}
+
+header li {
+  /* border: 2px solid rgb(119, 175, 87); */
+  width: 100%;
+}
+
+header a {
+  display: block;
+  color: white;
+  width: 100%;
+  height: 100%;;
+  padding-top: 1.5rem;
+  cursor: pointer;
+  /* text-align: center; */
+}
+
+header li:hover {
+  opacity: 0.4;
+}
+
+.list {
+  padding-left: 0.6rem;
+}
+
+.list2 {
+  margin-right: 0.3rem;
+}
+
+/*----------------
+laptop
+-----------------*/
+@media(min-width: 620px) {
+  header {
+    display: none;
+  }
+  
+  body {
+    background-color:rgba(255,255,255, 0.0);
+    background-blend-mode: normal;
+  }
+}
+
+
+
```

Comment:
you should leave comments about comment out

// FIXME
// all style will be applied in next project

Reply:
I deleted it!